### ISSUE TITLE
Keypoint RCNN

### ISSUE THREAD

**Issue by songkevinc** [youngwanLEE/vovnet-detectron2#17]
Hi,

Thanks for this great extension. I'm currently looking to try different backbones for keypoint detection in detectron2. I have to say I'm sort of loss on how to go about replacing a backbone on an existing Keypoint RCNN architecture. Do you have any Vovnet implemented in Keypoint RCNN? Also, I'm looking for a good tutorial/example for swapping backbone architecture in Detectron2. I'd appreciate any help. Thank you!

**Comment by VGrondin** [comment]
Did you make it work? I am interested in doing the same.

**Comment by songkevinc** [comment]
Ha, I did eventually get it to work a few months ago. I ended up ditching the project because I didn't see any significant improvements.

**Comment by VGrondin** [comment]
Thanks for answering and that's disappointing. Do you mind explaining/showing a bit of code on how you made it work?

**Comment by songkevinc** [comment]
Yeah, I'm a little swamped at work at the moment, but I'll try to go dig up my unorganized code and post something in the next few days, haha.

**Comment by VGrondin** [comment]
Alright looking forward to it hahah
